tests/test_reportportal_endpoint.py

```
@@ -574,3 +1187,409 @@ def test_capabilities_includes_rp_enabled(self, _rp_enabled_env):
+        resp.text = text
+        resp.json.return_value = json_return
+        exc = Exception("boom")
+        exc.response = resp
+        return exc
+
+    def test_dict_body_extracts_message_field(self, _rp_enabled_env):
+        """When RP returns a JSON dict with 'message', extract it for user."""
+        from jenkins_job_insight.main import _rp_error_message
+
+        exc = self._make_exc_with_response(json_return={"message": "Token expired"})
+        user_msg, log_msg = _rp_error_message(exc, "finding launch")
```

Ruff (0.15.10)

[warning] Unpacked variable `log_msg` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)